pkgs/top-level/all-packages.nix

```diff
@@ -6513,9 +6513,9 @@ in
   yaft = callPackage ../applications/misc/yaft { };
 
   yarn = callPackage ../development/tools/yarn  { };
-
-  yarn2nix = callPackage ../development/tools/yarn2nix { };
-  inherit (yarn2nix) mkYarnPackage;
+  
+  yarn2nix = throw "Use upstream https://github.com/moretea/yarn2nix";
```

Comment:
for `yan2nix` binary, why not using

```
  yarn2nix = (callPackage (fetchFromGitHub {
    owner = "moretea";
    repo = "yarn2nix";
    rev = "master";
    sha256 = "1f83cr9qgk95g3571ps644rvgfzv2i4i7532q8pg405s4q5ada3h";
  }) {}).yarn2nix;
```

Reply:
1. This doesn't lock down the revision, so this will break as soon as something new is pushed to Yarn2nix's `master`.
2. Nixpkgs doesn't allow importing from the output of derivations (such as fetches), since Hydra doesn't support that.